pkgs/development/libraries/geos/default.nix

```diff
@@ -1,22 +1,33 @@
-{ lib, stdenv, fetchurl }:
+{ lib
+, stdenv
+, fetchurl
+, cmake }:
 
 stdenv.mkDerivation rec {
   pname = "geos";
-  version = "3.9.1";
+  version = "3.10.2";
 
   src = fetchurl {
     url = "https://download.osgeo.org/geos/${pname}-${version}.tar.bz2";
-    sha256 = "sha256-fmMFB9ysncB1ZdJJom8GoVyfWwxS3SkSmg49OB1+OCo=";
+    sha256 = "sha256-ULvFmaw4a0wrOWLcxBHwBAph8gSq7066ciXs3Qz0VxU=";
   };
 
+  buildInputs = [ cmake ];
+
   enableParallelBuilding = true;
 
-  # https://trac.osgeo.org/geos/ticket/993
-  configureFlags = lib.optional stdenv.isAarch32 "--disable-inline";
```

Comment:
question: for `geos39` do we want to keep this line ?

Reply:
Good catch. Per https://github.com/libgeos/geos/issues/512 I think we should leave it in for now.

Reply:
Updated with the latest force push.